packages/google-pay-integration/e2e/GooglePayBraintreeMockingResponses.ts

```diff
@@ -0,0 +1,72 @@
+/*
+INFO: clientToken generated from the following object without any security data for correct initialization of the braintree client
+
+const clientConfig = {
+    version: 2,
+    authorizationFingerprint: 'authorizationFingerprint',
+    configUrl:
+        'https://api.sandbox.braintreegateway.com:443/merchants/merchant_numeric_id/client_api/v1/configuration',
+    merchantAccountId: 'bigcommerce',
+    graphQL: {
+        url: 'https://payments.sandbox.braintree-api.com/graphql',
+        date: '2018-05-08',
+        features: ['tokenize_credit_cards'],
+    },
+    clientApiUrl:
+        'https://api.sandbox.braintreegateway.com:443/merchants/merchant_numeric_id/client_api',
+    environment: 'sandbox',
+    merchantId: 'merchant_numeric_id',
+    assetsUrl: 'https://assets.braintreegateway.com',
+    authUrl: 'https://auth.venmo.sandbox.braintreegateway.com',
+    venmo: 'off',
+    challenges: ['cvv', 'postal_code'],
+    threeDSecureEnabled: true,
+    analytics: {
+        url: 'https://origin-analytics-sand.sandbox.braintree-api.com/merchant_numeric_id',
+    },
+    paypalEnabled: true,
+    paypal: {
+        billingAgreementsEnabled: true,
+        environmentNoNetwork: false,
+        unvettedMerchant: false,
+        allowHttp: true,
+        displayName: 'Bigcommerce',
+        clientId: 'clientId',
+        baseUrl: 'https://assets.braintreegateway.com',
+        assetsUrl: 'https://checkout.paypal.com',
+        directBaseUrl: null,
+        environment: 'offline',
+        braintreeClientId: 'braintreeClientId',
+        merchantAccountId: 'bigcommerce',
+        currencyIsoCode: 'USD',
+    },
+};
+
+const clientToken = btoa(unescape(encodeURIComponent(clientConfig)))
+ */
```

Comment:
What's this giant comment block for?

Reply:
To understand what a clientToken is

Reply:
I am afraid talking about token generation openly is not a good idea.
Could you please move it to the internal confluence and link to it here?
The rest of the PR looks good to me.

Reply:
@bc-peng done